x/ccv/consumer/ibc_module.go

```diff
@@ -72,41 +73,41 @@ func (am AppModule) OnChanOpenInit(
 // validateCCVChannelParams validates a ccv channel
 func validateCCVChannelParams(
 	ctx sdk.Context,
-	keeper keeper.Keeper,
+	ccvkeeper keeper.Keeper,
 	order channeltypes.Order,
 	portID string,
 	version string,
 ) error {
 	// Only ordered channels allowed
 	if order != channeltypes.ORDERED {
-		return sdkerrors.Wrapf(channeltypes.ErrInvalidChannelOrdering, "expected %s channel, got %s ", channeltypes.ORDERED, order)
+		return errorsmod.Wrapf(channeltypes.ErrInvalidChannelOrdering, "expected %s channel, got %s ", channeltypes.ORDERED, order)
 	}
 
 	// the port ID must match the port ID the CCV module is bounded to
-	boundPort := keeper.GetPort(ctx)
+	boundPort := ccvkeeper.GetPort(ctx)
 	if boundPort != portID {
-		return sdkerrors.Wrapf(porttypes.ErrInvalidPort, "invalid port: %s, expected %s", portID, boundPort)
+		return errorsmod.Wrapf(porttypes.ErrInvalidPort, "invalid port: %s, expected %s", portID, boundPort)
 	}
 
 	// the version must match the expected version
 	if version != types.Version {
-		return sdkerrors.Wrapf(types.ErrInvalidVersion, "got %s, expected %s", version, types.Version)
+		return errorsmod.Wrapf(types.ErrInvalidVersion, "got %s, expected %s", version, types.Version)
 	}
 	return nil
 }
 
 // OnChanOpenTry implements the IBCModule interface
-func (am AppModule) OnChanOpenTry(
-	ctx sdk.Context,
-	order channeltypes.Order,
-	connectionHops []string,
-	portID,
-	channelID string,
-	chanCap *capabilitytypes.Capability,
-	counterparty channeltypes.Counterparty,
-	counterpartyVersion string,
+func (AppModule) OnChanOpenTry(
+	_ sdk.Context,
```

Comment:
Please put formatting changes in a separate PR. We merged Notional's linting and formatting PRs a couple weeks ago. Why wasn't this changed then?